### PR TITLE
Need test to verify switches can match IP_DSCP field

### PR DESCRIPTION
**Pull request by brandonchau-github**
Locally ran test:
```
adding test FaucetDscpRewriteTest

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
running 0 tests in parallel and 1 tests serial
test_untagged (mininet_tests.FaucetDscpRewriteTest) ... *** Error setting resource limits. Mininet's performance may be affected.
ok

----------------------------------------------------------------------
Ran 1 test in 29.534s

OK


test results
============
--- {}
...
```

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -897,8 +897,9 @@ Each acl contains a list of rules: a packet will have the first matching rule
 applied to it.
 
 Each rule is a dictionary containing the single key 'rule' with matches
-and actions. Matches are key/values based on the ryu RESTFul API. Actions
-is a dictionary of actions to apply upon match.
+and actions. Matches are key/values based on the ryu RESTFul API.
+https://ryu.readthedocs.io/en/latest/app/ofctl_rest.html#reference-description-of-match-and-actions
```
Comment (gizmoguy) [inline docs/configuration.rst]: We could use an RST hyperlink here to make this a bit cleaner, e.g:

`ryu RESTFul API
<https://ryu.readthedocs.io/en/latest/app/ofctl_rest.html#reference-description-of-match-and-actions>`_

File: tests/integration/mininet_tests.py [hunk 2]
```diff
@@ -6755,6 +6755,76 @@ def test_switching(self):
             source_host, overridden_host, rewrite_host, overridden_host)
 
 
+class FaucetDscpRewriteTest(FaucetUntaggedTest):
```
Comment (gizmoguy) [inline tests/integration/mininet_tests.py]: This is technically not a `FaucetDscpRewriteTest` test, it's a `FaucetDscpMatchTest` or `FaucetDscpMatchEthDstRewriteTest`

**Comment by gizmoguy** [comment]
Apart from those two minor changes LGTM. Push up a fix for both of those and I'll approve for merging.

File: docs/configuration.rst (hunk 1 above)
Comment (brandonchau-github) [reply docs/configuration.rst]: Done!

File: tests/integration/mininet_tests.py (hunk 2 above)
Comment (brandonchau-github) [reply tests/integration/mininet_tests.py]: Done!

File: docs/configuration.rst [hunk 3]
```diff
@@ -897,8 +897,9 @@ Each acl contains a list of rules: a packet will have the first matching rule
 applied to it.
 
 Each rule is a dictionary containing the single key 'rule' with matches
-and actions. Matches are key/values based on the ryu RESTFul API. Actions
-is a dictionary of actions to apply upon match.
+and actions. Matches are key/values based on the ryu RESTFul API.
+<https://ryu.readthedocs.io/en/latest/app/ofctl_rest.html#reference-description-of-match-and-actions>
```
Comment (gizmoguy) [inline docs/configuration.rst]: Syntax is still a little wrong here, you want to use the following exactly (RST is a bit weird for URLs):

```
`ryu RESTFul API
<https://ryu.readthedocs.io/en/latest/app/ofctl_rest.html#reference-description-of-match-and-actions>`_

Check out the embeddable URI documentation here: https://docutils.sourceforge.io/docs/user/rst/quickref.html#external-hyperlink-targets
```

Comment (brandonchau-github) [reply docs/configuration.rst]: My bad, updated this and did a manual inspection with the python previewer.  This should be corrected in the new patch.

**Pull request merged**